UPGRADE.rst

```diff
@@ -88,6 +88,14 @@ for example:
 Upgrading to v1.32.0
 ====================
 
+Dropping support for end-of-life Python, Postgres and SQLite versions
+---------------------------------------------------------------------
+
+In line with our `deprecation policy <https://github.com/matrix-org/synapse/blob/v1.32.0rc1/docs/deprecation_policy.md>`_,
```

Comment:
Maybe want to change this link so it points to the stable release (or master) instead of the RC?

Reply:
Probably best to keep it at the current stable just in case we change it significantly (or move it).